packages/core/deploy/005_deploy_sink_oracle.js

```diff
@@ -25,5 +25,5 @@ const func = async function (hre) {
   });
 };
 module.exports = func;
-func.tags = ["SinkOracle", "bridge-l2"];
+func.tags = ["SinkOracle", "l2-chainbridge"];
```

Comment:
it's not called `chainbridge` any more, is it?

Reply:
it is still yeah